mlflow/pyfunc/backend.py

```diff
@@ -19,8 +19,9 @@ class PyFuncBackend(FlavorBackend):
     """
         Flavor backend implementation for the generic python models.
     """
-    def __init__(self, config, no_conda=False, install_mlflow=False, **kwargs):
+    def __init__(self, config, workers=1, no_conda=False, install_mlflow=False, **kwargs):
```

Comment:
Why is the default (one) here different from in the `models` CLI (four)? There's probably a good reason, I was just wondering

Reply:
Actually I think we might not need the default at all here?